src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/InstallPluginsStep.java

```diff
@@ -54,7 +54,7 @@ public void execute(InstanceConfiguration config)
                     .addArgument("--batch")
                     .addArgument(plugin.getUri());
             
-            ProcessUtil.executeScript(config, cmd, environment, null);
+            ProcessUtil.executeScript(config, cmd, ProcessUtil.createEnvironment(environment), null);
```

Comment:
I don't understand this change either. If you want those vars to be removed from the env, that's is done anyway (for ProcessUtil.executeScript always calls ProcessUtils.createEnvironment to build the env).